plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -472,8 +472,8 @@ class MarathonStore extends GetSetBaseStore {
     const groups = new ServiceTree(data);
 
     const apps = groups.reduceItems(function(map, item) {
```

Comment:
This whole reduce function is so confusing. Maybe we could write some comments to help explain why we reduce groups to frameworks but then call them apps 😢